src/d3a/d3a_core/export.py

```diff
@@ -380,13 +425,11 @@ def plot_all_unmatched_loads(self):
         """
         Plot unmatched loads of all loads in the configuration into one plot
         """
-        unmatched_key = 'deficit [kWh]'
-        data = list()
         root_name = self.area.slug
-        title = 'Unmatched Loads for all devices in {}'.format(root_name)
-        xtitle = 'Time'
-        ytitle = 'Energy (kWh)'
-        barmode = 'stack'
+        plot_desc = PlotDescription(data=[], barmode="stack",
```

Comment:
One nit and one more important comment.
Nit:
```suggestion
        plot_description = PlotDescription(data=[], barmode="stack",
```
These lines seem not to be tested. Hannes said that the exporter should be deprecated; if this will happen soon, then you can ignore it. If it won't happen soon, then I'd suggest to write tests.

Reply:
https://github.com/gridsingularity/d3a/pull/1219/commits/6c441c8932252514d0f7e5bbcb0672ec2f390395